vortex-python/benchmark/test_scan.py

```diff
@@ -0,0 +1,95 @@
+import duckdb
+import os
+import polars as pl
+import pyarrow as pa
+import pytest
+import vortex as vx
+from vortex.expr import column
+
+
+@pytest.fixture(scope="session")
+def vxf(tmpdir_factory) -> vx.VortexFile:  # pyright: ignore[reportUnknownParameterType, reportMissingParameterType]
```

Comment:
and this is probably `pytest.TempPathFactory`

Reply:
Nice! Also fixed.